libraries/AP_DroneCAN/AP_DroneCAN.cpp

```diff
@@ -710,10 +722,15 @@ void AP_DroneCAN::SRV_send_esc_hobbywing(void)
 
         for (uint8_t i = esc_offset; i < max_esc_num && k < 20; i++) {
             if ((((uint32_t) 1) << i) & _ESC_armed_mask) {
-                // TODO: ESC negative scaling for reverse thrust and reverse rotation
-                float scaled = cmd_max * (hal.rcout->scale_esc_to_unity(_SRV_conf[i].pulse) + 1.0) / 2.0;
+                float scaled = 0;
```

Comment:
Hobbywing seems to have their own duplicate of the functionality for sending RawCommands. I added identical support to that function as well while I was in here.